src/cli/argv_parser.ts

```diff
@@ -168,6 +170,16 @@ const ArgvParser = {
         'Use predictable ids in messages (option ignored if using parallel)',
         false
       )
+      .option(
+        '--publish',
+        'Publish a report to https://reports.cucumber.io',
+        false
+      )
+      .option(
+        '--publish-quiet',
```

Comment:
Note, I think this is named wrong. This initially makes me think it suppresses output when using the publish option instead of suppressing an advertisement about publish. Given already used in a ruby/jvm this is fine for now but in long run would prefer to rename this to something like `--publish-suppress-advertisement`

Reply:
Yes we can change it in the future but then we'll need to do so in all implementations.